test/ibmq/test_experiment.py

```diff
@@ -422,6 +423,7 @@ def test_upload_analysis_result(self):
                                     result_type=ref_result.type,
                                     device_components=[device_comp.type],
                                     quality='No Information',
+                                    verified=False,
```

Comment:
linter is complaining about this:

https://github.com/Qiskit/qiskit-ibmq-provider/pull/909/checks?check_run_id=2240623402#step:5:10

I'm not sure how valid it is, I've had to bypass some lint failures in the test code before, maybe try to recreate locally and see if you can make it happy (I create a virtual environment and then use the make file to run things like pylint, mypy, etc since tox isn't really used for those).